docs/cli.md

```diff
@@ -616,23 +616,25 @@ viam logout
 
 ### `module`
 
-The `module` command allows to you to manage custom {{< glossary_tooltip term_id="module" text="modules" >}}
+The `module` command allows to you to manage custom {{< glossary_tooltip term_id="module" text="modules" >}}.
 This includes:
 
-- Creating a new custom {{< glossary_tooltip term_id="resource" text="modular resource" >}}
+- Generating stub files for a new module
+- Creating metadata for a new custom {{< glossary_tooltip term_id="resource" text="modular resource" >}}
 - Uploading a new module to the [Viam registry](https://app.viam.com/registry)
 - Uploading a new version of your module to the [Viam registry](https://app.viam.com/registry)
 - Updating an existing module in the Viam registry
 - Updating a module's metadata file based on models it provides
 - Building your module for different architectures using cloud runners
 - Building a module locally and running it on a target device. Rebuilding & restarting if already running.
 
-See [Upload a custom module](/how-tos/create-module/#upload-your-module-to-the-modular-resource-registry) and [Update an existing module](/how-tos/manage-modules/#update-an-existing-module) for a detailed walkthrough of the `viam module` commands.
+See [Upload a custom module](/how-tos/upload-module/) and [Update an existing module](/how-tos/manage-modules/#update-an-existing-module) for a detailed walkthrough of the `viam module` commands.
```

Comment:
It's not a walkthrough of the commands - more a walkthrough of the process the commands are used for
```suggestion
See [Upload a custom module](/how-tos/upload-module/) and [Update an existing module](/how-tos/manage-modules/#update-an-existing-module) for more information.
```